modules/ROOT/pages/configure-client-management-azure.adoc

```diff
@@ -0,0 +1,70 @@
+= Configure Azure Client Management
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can use dynamic client registration to configure Azure client management with Anypoint Platform.
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+include::partial$include-nav-client-providers.adoc[]
+
+. Click *Add Client Provider*, and then select *OpenID Connect DCR for Azure*.
++
+The *Add OIDC Azure client provider* page appears.
++
+. Enter a name and description for your client provider.
+. Enter the following values from your identity provider’s configuration:
+.. *Issuer*: URL that the OpenID provider asserts is its trusted issuer.
+.. *Tenant ID for Azure AD*: The tenant ID from Azure Active Directory. For more information on obtaining your tenant ID, see https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenant[How to find your Azure Active Directory tenant ID].
+.. *Authorization Header*: The header to provide credentials to authenticate with a server. This header is required if the provider restricts registration requests to authorized clients.
+. Expand the *Advanced Settings* section. The following selections are optional:
+.. *Disable server certificate validation*: Disables server certificate validation if your Azure client management instance presents a self-signed certificate, or one signed by an internal certificate authority.
+.. *Enable client deletion in Anypoint Platform*: Enables deletion of clients created with this integration.
+.. *Enable client deletion and updates in IdP*: To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform to `{clientRegistrationUrl}/{clientID}`. +
+  * The `client_id` request parameter that is passed is the same as the `client_id` passed in the `PUT` request payload.
+  * The `token_endpoint_auth_method` passed in the payload is always `client_secret_basic`.
+  * Only the `Client Registration URL` and `Authorization Header` come from what is configured in Azure *Client Provider*.
```

Comment:
Only the `Authorization Header` come from what is configured in Azure *Client Provider*.